android/java/org/chromium/chrome/browser/rewards/tipping/TippingVerifiedCreatorToolTip.java

```diff
@@ -0,0 +1,119 @@
+        View contentView = inflater.inflate(R.layout.tipping_verified_creator_tooltip, null, false);
+        setLearnMore(contentView);
+        mPopupWindow = new PopupWindow(context);
+        mPopupWindow.setContentView(contentView);
+        mPopupWindow.setTouchable(true);
+        mPopupWindow.setFocusable(true);
+        mPopupWindow.setBackgroundDrawable(AppCompatResources.getDrawable(
+                context, R.drawable.tipping_verified_creator_tooltip_background));
+        mPopupWindow.setElevation(60);
+        mPopupWindow.setWidth((int) convertDPtoPX(context, 280.0f));
+        mPopupWindow.setTouchInterceptor((v, event) -> {
+            if (event.getAction() == MotionEvent.ACTION_OUTSIDE) {
+                mPopupWindow.dismiss();
+                return true;
+            }
+            return false;
+        });
+    }
+
+    private void setLearnMore(View view) {
+        TextView descriptionTextView = view.findViewById(R.id.verified_creator_description);
+        String descriptionText =
+                String.format(view.getResources().getString(R.string.tipping_tooltip_description),
+                        view.getResources().getString(R.string.learn_more));
+        SpannableString spannableLearnMore =
+                learnMoreSpannableString(view.getContext(), descriptionText);
+        descriptionTextView.setMovementMethod(LinkMovementMethod.getInstance());
+        descriptionTextView.setText(spannableLearnMore);
+    }
+
+    private SpannableString learnMoreSpannableString(Context context, String text) {
+        Spanned textToAgree = BraveRewardsHelper.spannedFromHtmlString(text);
+
+        SpannableString ss = new SpannableString(textToAgree.toString());
+
+        ClickableSpan clickableSpan = new ClickableSpan() {
+            @Override
+            public void onClick(@NonNull View textView) {
+                CustomTabActivity.showInfoPage(context, NEW_SIGNUP_DISABLED_URL);
+            }
+            @Override
+            public void updateDrawState(@NonNull TextPaint ds) {
+                super.updateDrawState(ds);
+                ds.setUnderlineText(false);
+            }
+        };
+        int learnMoreIndex = text.indexOf(context.getResources().getString(R.string.learn_more));
+
+        ss.setSpan(clickableSpan, learnMoreIndex,
+                learnMoreIndex + context.getResources().getString(R.string.learn_more).length(),
+                Spanned.SPAN_EXCLUSIVE_EXCLUSIVE);
+
+        ForegroundColorSpan foregroundSpan = new ForegroundColorSpan(
+                context.getResources().getColor(R.color.brave_rewards_modal_theme_color));
+        ss.setSpan(foregroundSpan, learnMoreIndex,
+                learnMoreIndex + context.getResources().getString(R.string.learn_more).length(),
+                Spanned.SPAN_EXCLUSIVE_EXCLUSIVE);
+        return ss;
+    }
+
+    private float convertDPtoPX(Context context, float dip) {
```

Comment:
use an upstream function to convert dp to px

Reply:
updated